zenfolio-test.customsubdomain.json

```diff
@@ -0,0 +1,19 @@
+{
+    "providerId": "Zenfolio",
```

Comment:
AssertionError: File name zenfolio-test.customsubdomain.json not matching the pattern. Expected zenfolio.customsubdomain-test.json

Reply:
file was renamed to match expected pattern.